spec/integration/spendings_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Authentication', type: :request do
+  let(:user) { User.new(name: 'Bilal Ahmed', email: 'bilal@gmail.com', password: 'password') }
+
+  before do
+    sign_in user
+  end
+
+  it 'Should show categories for sign in users' do
+    get categories_path
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'Should show new categories page' do
+    get new_category_path
+    expect(response).to have_http_status(:success)
+  end
+end
```

Comment:
- Good job done so far, but you have not properly done your integration tests. Remember integration test should ensure that your rendered template contains the right information filled in from the database. Please fix this 👍. You can use this [article](https://semaphoreci.com/community/tutorials/integration-testing-ruby-on-rails-with-minitest-and-capybara) as a reminder.